src/explorer/FolderTreeItem.ts

```diff
@@ -38,8 +38,9 @@ export class FolderTreeItem implements IAzureParentTreeItem {
     public async loadMoreChildren(node: IAzureNode<FolderTreeItem>): Promise<IAzureTreeItem[]> {
         const webAppClient: WebSiteManagementClient = nodeUtils.getWebSiteClient(node);
         const kuduClient: KuduClient = await this.siteWrapper.getKuduClient(webAppClient);
-
-        const fileList = await kuduClient.vfs.getItem(this.folderPath);
+        const httpResponse: kuduIncomingMessage = <kuduIncomingMessage>(await kuduClient.vfs.getItemWithHttpOperationResponse(this.folderPath)).response;

```

Comment:
(This probably should have been asked a while ago) Can this line throw exceptions? If an exception was thrown, can it be handled more gracefully than just letting VSC catching it?

Reply:
Kudu returns an error that bubbles up and is displayed to the user if the API call fails.  Do you think it'd be helpful to create an error node if it throws an exception?

Reply:
It should be fine if there is a way for users to retry - either by clicking the load more again or explicitly a retry button on a message bar. I am a bit afraid of raw error from network being displayed on UI won't be very readable.